.travis.yml

```diff
@@ -1,16 +1,38 @@
 language: elixir
-elixir:
-  - 1.8.2
+matrix:
+  include:
+    - elixir: "1.5.3"
+      otp_release: "18.3"
+    - elixir: "1.5.3"
+      otp_release: "19.3"
+    - elixir: "1.5.3"
+      otp_release: "20.3"
+    - elixir: "1.6.6"
+      otp_release: "19.3"
+    - elixir: "1.6.6"
+      otp_release: "20.3"
+    - elixir: "1.6.6"
+      otp_release: "21.3"
+    - elixir: "1.7.4"
+      otp_release: "19.3"
+    - elixir: "1.7.4"
+      otp_release: "22.3"
+    - elixir: "1.8.2"
+      otp_release: "20.3"
+    - elixir: "1.8.2"
+      otp_release: "22.3"
+    - elixir: "1.9.4"
+      otp_release: "20.3"
+    - elixir: "1.9.4"
+      otp_release: "22.3"
+    - elixir: "1.10.3"
+      otp_release: "21.3"
+    - elixir: "1.10.3"
+      otp_release: "22.3"
```

Comment:
Can we please skip the middle versions? Latest and earliest OTP for each Elixir version should be enough. Thank you!

Reply:
You are referring to 1.5.3 × 19.3 and 1.6.6 x 20.3, right?

Reply:
If so, done in d861b000f181e85a8c8977febece446c32d07a9c